src/component/archive/ArchiveTable/FeatureArchivedCell/FeatureArchivedCell.tsx

```diff
@@ -0,0 +1,36 @@
+import { VFC } from 'react';
+import TimeAgo from 'react-timeago';
+import { Tooltip, Typography } from '@mui/material';
+import { formatDateYMD } from 'utils/formatDate';
+import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+
+interface IFeatureArchivedCellProps {
+    value?: string | Date | null;
+}
+
+export const FeatureArchivedCell: VFC<IFeatureArchivedCellProps> = ({
+    value: archivedAt,
+}) => {
+    const { locationSettings } = useLocationSettings();
+
+    if (!archivedAt) return <TextCell />;
+
+    return (
+        <TextCell>
+            {archivedAt && (
+                <Tooltip
+                    title={`Archived on: ${formatDateYMD(
```

Comment:
I think we removed `on` from tooltips, so maybe this should be `Archived:` instead, but @NicolaeUnleash can help here.

Reply:
It was like this in my design, i forgot to update. But this will be review and aligned if we implement "X days ago" as a general solution (then we will decide how the tooltips to be)